### DISCUSSION TITLE
Why doesnt Servo provide builds for 32 bit systems?

### DISCUSSION THREAD

**Discussion by vixalien** [servo/servo#28178] [Q&A] [answered]
Hey why doesn't Servo provide builds for 32 bit systems as I see nothing that should be preventing this.

**Comment by jdm** [comment, marked as answer]
They're relatively rare, take extra resources to produce, and none of our developers have access to such a system to test the resulting builds.

**Comment by vixalien** [comment]
Well it would still be nice to have them because it's 2021 and I still have
a 32bit PC. And it would serve as a fallback because you know most 32bit
programs can run on 64bit infrastructure but the inverse is impossible.

I would like to try Servo, but building it for myself (probably gonna take
2hrs+) doesn't sound worth it.

If there is a cloud build system, is it allowed to implement a build step
that provide the relevant 32bit builds?

**Reply by jdm** [reply]
I do not believe we would accept a PR to add the extra nightly builds. The audience for developer nightly builds is too small to make it worthwhile.

**Reply by vixalien** [reply]
Okay understood now. I just thought Servo on 32bit was impossible or complicated, but its a mere question of audience.

Thanks
